tree/30_generic_methods/service_stop.cf

```diff
@@ -39,14 +39,25 @@ bundle agent service_stop(service_name)
     "class_prefix"      string => canonify(join("_", "promisers"));
     "args"                     slist => { "${service_name}" };
 
+  class:
+
+    "is_inner_method" expression => "!${report_data.should_report}";
+
   methods:
+    !is_inner_method::
```

Comment:
why should service_action ever make reports here?

Reply:
In fact the condition is only needed on the end side we should always disable inner reporting, if it is already an inner method